annotation/src/main/java/com/android/designcompose/annotation/Builder.kt

```diff
@@ -20,11 +20,16 @@ package com.android.designcompose.annotation
  * Generate an interface that contains functions to render various nodes in a Figma document
  *
  * @param id the id of the Figma document. This can be found in the url, e.g. figma.com/file/<id>
- * @param version a version string that gets written to a generated JSON file used for the Design
- *   Compose Figma plugin
+ * @param designVersion version id of the Figma document.
+ * @param composeVersion a version string that gets written to a generated JSON file used for the
+ *   Design Compose Figma plugin
  */
 @Target(AnnotationTarget.CLASS)
-annotation class DesignDoc(val id: String, val version: String = "0")
+annotation class DesignDoc(
+    val id: String,
+    val designVersion: String = "",
+    val composeVersion: String = "0"
```

Comment:
I might change this to `interfaceVersion`, because I think the `@DesignDoc` stuff is defining the interface between the code and the design doc.

Reply:
Added a comment to the discussion above.